src/params/progression.jl

```diff
@@ -50,7 +52,13 @@ function sample_severity(rng::AbstractRNG, age::Real)
   else;             dist = severity_dists[6]
   end
 
-  rand(rng, dist) |> Severity
+  severity = rand(rng, dist) |> Severity
+  severity_int = severity |> UInt8
+  if severity > Asymptomatic && vaccinated(immunity) && rand(rng) < vaccination_effectiveness[severity_int]
+    severity_int = severity_int - 0x01
```

Comment:
what does that stand for?

Reply:
for vaccinated / immune people  `Critical->Severe , Severe->Mild, Mild->Asymptomatic` with some probability. I merged 3 ifs to one

Reply:
OK, understood. Please add a comment for this ;)

Reply:
ok, I'll add it tomorrow ;)